plugins/modules/system/sap_task_list_execute.py

```diff
@@ -54,10 +54,10 @@
   task_parameters:
     description:
       - The tasks and the parameters for execution.
-      - If the task list do not need any parameters. This could be empty.
-      - If only specific tasks from the task list should be executed.
-        The tasks even when no parameter is needed must be provided.
-        Alongside with the module parameter I(task_skip=true).
+      - If the task list does not need any parameters, this could be empty.
+      - If only specific tasks from the task list should be executed,
+        the tasks even when no parameter is needed must be provided
+        alongside with the module parameter I(task_skip=true).
```

Comment:
```suggestion
      - If the task list does not need any parameters, this could be empty.
      - If only specific tasks from the task list should be executed,
        the tasks even when no parameter is needed must be provided
        alongside with the module parameter I(task_skip=true).
```
I'm not sure if this is implied, so let's check